Automattic/woocommerce-payments
# feat: WooPay button SSR

#### Changes proposed in this Pull Request









Adding some SSR to pre-render the WooPay button on non-blocks pages.

This is useful to ensure that the page has less of a jarring effect on first load.
The SSR honors the theme/size of the button, but not its contents. The SSR also loads the button in a `disabled` state.


Before:
![2023-10-12 10 05 20](https://github.com/Automattic/woocommerce-payments/assets/273592/3aa4b8fe-bf5d-4d7a-9373-47814cd4beb5)
![2023-10-12 10 04 40](https://github.com/Automattic/woocommerce-payments/assets/273592/8cd98c10-2276-4740-b3e1-08b5fce1b465)

After:
![2023-10-13 10 18 01](https://github.com/Automattic/woocommerce-payments/assets/273592/b2a42d9f-91f7-4d4a-998f-ac78bca19606)
![2023-10-12 10 10 00](https://github.com/Automattic/woocommerce-payments/assets/273592/cb00888d-ccbe-4387-87d8-207ae160c228)
![2023-10-12 10 09 06](https://github.com/Automattic/woocommerce-payments/assets/273592/a08daab3-c055-4f2b-94e6-9d5a95003b4e)


#### Testing instructions





- Add the WooPay button to any of the pages (product, cart, checkout)
- Load the shortcode version of the pages
- The WooPay button should be pre-rendered in a `disabled` state
- Theme/Size should be honored during the SSR

-------------------


**Post merge**
